npm/pkg/dataplane/policies/policymanager_linux.go

```diff
@@ -174,12 +181,14 @@ func (pMgr *PolicyManager) deleteJumpRule(policy *NPMNetworkPolicy, direction Un
 	}
 
 	specs = append([]string{baseChainName}, specs...)
+	timer := metrics.StartNewTimer()
 	errCode, err := pMgr.runIPTablesCommand(util.IptablesDeletionFlag, specs...)
+	metrics.RecordIPTablesDeleteLatency(timer)
 	// if this actually happens (don't think it should), could use ignoreErrorsAndRunIPTablesCommand instead with: "Bad rule (does a matching rule exist in that chain?)"
-	if err != nil && errCode != doesNotExistErrorCode {
+	if err != nil && errCode != doesNotExistErrorCode && errCode != couldntLoadTargetErrorCode {
```

Comment:
add couldntLoadTargetErrorCode in case the netpol chain we're deleting from somehow doesn't exist